src/alert/lsst.rs

```diff
@@ -462,6 +480,64 @@ pub struct LsstAlert {
     pub cutout_template: Option<Vec<u8>>,
 }
 
+fn flux2mag(flux: f32, flux_err: f32) -> (f32, f32, f32) {
+    // convert from nJy to Jy
+    let flux = flux.abs() * 1e-9;
+    let flux_err = flux_err * 1e-9;
+
+    // convert from Jy to AB mag
+    let mag = -2.5 * (flux).log10() + 8.9;
+    let sigma = 1.0857362047581294 * (flux_err / flux);
+    let diffmaglim = -2.5 * (5.0 * flux_err).log10() + 8.9;
```

Comment:
This looks like a duplication of logic with the `fluxerr2diffmaglim` function below. I would either replace this with a call to `fluxerr2diffmaglim` or (my preference) remove this line and having this function return just `(mag, sigma)` and call `fluxerr2diffmaglim` separately when you want a value for `diffmaglim`.

Reply:
removed in the latest commit